Holovibes/sources/io/output_file/output_avi_file.cc

```diff
@@ -34,6 +35,32 @@ void OutputAviFile::write_header()
     }
 }
 
+static void move_frame(
+    uchar* output, const char* frame, const unsigned short width, const unsigned short height, const int byte_type)
+{
+    // compute ratio between width and height to know which dimension to fill
+    size_t width_ratio = std::max(width / height, 1);
+    size_t height_ratio = std::max(height / width, 1);
+    size_t add_byte_type = 0;
+    if (byte_type == camera::Endianness::LittleEndian)
+        add_byte_type = 1;
```

Comment:
You can combine this into a single line, using the result of the comparison as the value.
Btw, couldn't you use smth else than a size_t for something that is either 0 or 1 ? Like a boolean